### DISCUSSION TITLE
Enable comments on ads

### DISCUSSION THREAD

**Discussion by loftwah** [dailydotdev/daily#621] [Feature Requests]
I noticed an advertisement for the Shortcut platform when I opened a new tab, and I have recently started using it and found it to be quite great. It would be cool if we could also comment and vote on ads so those in the community can get a sense of if it is something worth actually looking into. I wouldn't interact with ads very often, but this is one that I would have but was unable to.

**Comment by idoshamun** [comment]
Hi @loftwah,

Thanks for this suggestion. I love it :)
